niap-cc/Permissions/Tester/app/src/main/java/com/android/certifications/niap/permissions/SignaturePermissionTester.java

```diff
@@ -577,45 +619,40 @@ public void onAvailable(Network network) {
                         } catch (ReflectiveOperationException e) {
                             throw new UnexpectedPermissionTestFailureException(e);
                         }
-                    } else {
+                    } else  if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
```

Comment:
The conditional above checks for an SDK level < Android S, should this conditional check for an SDK level >= S?

Reply:
Change the first conditional as <=Android S. Let me check it again...

Reply:
checked

Reply:
I thought the above block previously ran successfully on devices running Android R, but if this block works for Android R as well, then this change looks good to me.